NodeBase/entry_point.sh

```diff
@@ -25,6 +25,10 @@ if [ ! -z "$REMOTE_HOST" ]; then
   REMOTE_HOST_PARAM="-remoteHost $REMOTE_HOST"
 fi
 
+if [ ! -z "$FIREFOX_VERSION" ]; then
+  sudo ln -fs /opt/firefox/${FIREFOX_VERSION}/firefox /usr/bin/firefox
+fi
+
```

Comment:
This entry point isn't overridden in some of those, so I think I'd rather keep this here than complicate this further. I'll be opening a new pull request after rebasing from the recent changes shortly.